src/core/src/package_managers/ZypperPackageManager.py

```diff
@@ -758,13 +758,13 @@ def update_os_patch_configuration_sub_setting(self, patch_configuration_sub_sett
     def is_reboot_pending(self):
         """ Checks if there is a pending reboot on the machine. """
         try:
-            pending_file_exists = os.path.isfile(self.REBOOT_PENDING_FILE_PATH)     # not intended for zypper, but supporting as back-compat
```

Comment:
The changes in this file is only removing of spaces. These changes can now be reverted to remove this file from PR.
Same for the YumPackageManager.py file.

Reply:
This is merge code. for now  i will keep this.